modules/ROOT/pages/rtc/getting-started.adoc

```diff
@@ -10,7 +10,9 @@ include::partial$misc/requires_5_9v.adoc[]
 
 This procedure will assist with setting up {productname} with real-time collaboration.
 
-IMPORTANT: include::partial$misc/secure-context.adoc[]
+IMPORTANT: **
+include::partial$misc/secure-context.adoc[]
+**
```

Comment:
This conversion is incorrect as is makes all the content bold. Instead, we need to use a block admonition like so:
```suggestion
[IMPORTANT]
====
include::partial$misc/secure-context.adoc[]
====
```

Reply:
done